interfaces/cython/cantera/test/test_onedim.py

```diff
@@ -510,9 +508,9 @@ def test_save_restore(self):
 
         self.create_sim(p, Tin, reactants)
         self.solve_fixed_T()
-        filename = pjoin(self.test_work_dir, 'onedim-fixed-T.xml')
-        if os.path.exists(filename):
-            os.remove(filename)
+        filename = self.test_work_path / "onedim-fixed-T.xml"
+        if filename.is_file():
+            filename.unlink()
```

Comment:
```suggestion
        # In Python >= 3.8, this can be replaced by the missing_ok argument
        if filename.is_file():
            filename.unlink()
```
So that we have a note that this can be simplified in the future